src/main/java/newbank/server/Account.java

```diff
@@ -63,6 +65,15 @@ public Money getBalance() {
     return balance;
   }
 
+  public void moveMoneyToAccount(Account destination, Money amount) throws AccountBalanceInsufficientException {
+    if (balance.isLessThan(amount)) {
+      throw new AccountBalanceInsufficientException(amount, balance);
+    }
+
+    balance = balance.subtract(amount);
+    destination.credit(amount);
```

Comment:
Why not use Jules' `Account::credit` and `Account::debit`?

Reply:
Right, forgot about the debit. And `.credit` is being used here, right? :smile:

Reply:
Fixed.

Regards,
Jules